### ISSUE TITLE
Contract that was finished is back in conductor

### ISSUE THREAD

**Issue by rileystewart** [CityofPittsburgh/pittsburgh-purchasing-suite#68]
Guide rail was finished on 1/29.  Ed did not go into conductor to officially move it to finished until late last week.  He finished it, it updated in Scout, and was removed from conductor. Now it is back in conductor showing it is waiting for city contract complete.

**Comment by bsmithgall** [comment]
Please provide the item number and I'll look into it later today.

**Comment by rileystewart** [comment]
7095 is the spec number. I'm not sure what the item number would be.

**Comment by bsmithgall** [comment]
Is this a duplicate? Or are there supposed to be two separate contracts (with two separate spec numbers) for this? The other spec is 7532.

**Comment by rileystewart** [comment]
I removed the one labeled incorrectly. There is only one

Riley R. Stewart

**Comment by bsmithgall** [comment]
Great.

**Comment by rileystewart** [comment]
I removed the duplicate but it still doesn't answer why Guide Rail is still in there.  I watched the one labeled Guide Rail 7095 be finished in conductor by ed.  it finished on 1/29.  it is still showing up in conductor.  It was completed and the new one created should have been 7532, expiration 10-31-17.  This new one was created in Scout and 7095 disappeared....a day later it was back and not marked as complete
